dracut/30ignition/ignition-setup-user.service

```diff
@@ -10,12 +10,10 @@ Before=ignition-disks.service
 Before=ignition-files.service
 
 # On diskful boots, ignition-generator adds Requires/After on
-# dev-disk-by\x2dlabel-boot.device.
+# Requires=boot.mount
+# After=boot.mount
```

Comment:
Did you see the suggestions at https://github.com/coreos/ignition-dracut/pull/179#discussion_r418694131 ?

Reply:
Er, those are comments, but cruft. Removed.